internal/pkg/skuba/addons/addons.go

```diff
@@ -83,6 +84,7 @@ type Addon struct {
 	callbacks          addonCallbacks
 	addonPriority      addonPriority
 	getImageCallbacks  []getImageCallback
+	isCniAddon         bool
```

Comment:
Maybe it could be generalized as `addOnType AddOnType` and define 
```
type AddOnType string

const (
     CniAddOn AddOnType "CNI"
)
```
In this way, if in the future other types of addons are supported, it won't be necessary to maintain multiple flags, but add more

Reply:
Good idea, I'll make that change.